src/main/java/core/basesyntax/service/impl/TransactionProcessImpl.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.ReportDao;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.TransactionProcess;
+import core.basesyntax.strategy.StrategyFruitTransaction;
+import core.basesyntax.transaction.OperationHandler;
+
+public class TransactionProcessImpl implements TransactionProcess {
+    private final StrategyFruitTransaction strategyFruitTransaction;
+    private final ReportDao reportDao;
+
+    public TransactionProcessImpl(StrategyFruitTransaction strategyFruitTransaction,
+                                  ReportDao reportDao) {
+        this.strategyFruitTransaction = strategyFruitTransaction;
+        this.reportDao = reportDao;
+    }
+
+    @Override
+    public void process(FruitTransaction fruitTransaction) {
+        if (fruitTransaction.getOperation() == Operation.BALANCE) {
+            return;
+        }
```

Comment:
it make no sense and nothing to do

Reply:
without this check i have NPE, so i think it make sense